<launch>

    <node name="tracking_target_node" pkg="pantilt_control" type="tracking_target.py" output="screen">
        <param name="pan_joint_name"  value="yaw_joint"   />
        <param name="tilt_joint_name" value="pitch_joint" />
        <rosparam param="pan_joint_limit"> {min: -1.57, max: 1.57} </rosparam>
        <rosparam param="tilt_joint_limit"> {min: -1.57, max: 1.57} </rosparam>
        <param name="p_gain" value="0.075" />
        <param name="d_gain" value="0.5" /> <!-- no need in almost situations -->
        <param name="i_gain" value="0.0" /> <!-- no need in almost situations -->
        <param name="step_time" value="0.1" /> <!-- [sec] fast -> set small value -->
    </node>

    <node name="detect_target_color_node" pkg="pantilt_control" type="detect_target_color.py" output="screen">
        <rosparam param="hsv_mask1_lower"> [  0,  50,   0] </rosparam>
        <rosparam param="hsv_mask1_upper"> [ 10, 255, 255] </rosparam>
        <rosparam param="hsv_mask2_lower"> [160,  50,   0] </rosparam>
        <rosparam param="hsv_mask2_upper"> [180, 255, 255] </rosparam>
    </node>

</launch>
